tests/framework/extensions/rke1/componentchecks/etcdversion.go

```diff
@@ -5,24 +5,46 @@ import (
 	"strings"
 
 	"github.com/rancher/rancher/tests/framework/clients/rancher"
-	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	nodes "github.com/rancher/rancher/tests/framework/pkg/nodes"
 	"github.com/sirupsen/logrus"
 )
 
 // CheckETCDVersion will check the etcd version on the etcd node in the provisioned RKE1 cluster.
-func CheckETCDVersion(client *rancher.Client, nodes []*nodes.Node, nodeRoles []string) ([]string, error) {
+func CheckETCDVersion(client *rancher.Client, nodes []*nodes.Node, clusterID string) ([]string, error) {
+	steveClient, err := client.Steve.ProxyDownstream(clusterID)
+	if err != nil {
+		return nil, err
+	}
+
+	nodesList, err := steveClient.SteveType("node").List(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	client, err = client.ReLogin()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Does it need to re-login to list the nodes since the user's schema isn't changing?

Reply:
I don't think it does honestly, we already instantiate the proxy client

Reply:
Removed it, that's a good point